felles/src/main/kotlin/no/nav/familie/kontrakter/felles/ef/BehandlingMetode.kt

```diff
@@ -0,0 +1,7 @@
+package no.nav.familie.kontrakter.felles.ef
+
+enum class BehandlingMetode {
```

Comment:
Burde denne ligge i den samme filen som `BehandlingsstatistikkDto`? 
Skal den brukes til noe annet spesielt då den ligger direkt under felles/ef?

Reply:
Så for meg at den vil kunne brukes internt i fremtiden også for å skille på det, men jeg flytter det sammen med behandlingsstatistikk nå, så kan det heller refaktorere ved senere tid i såfall.